Fix LangChain package version number

Fix LangChain package version number. (latest version is 0.2.5)

Run npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for langchain@^1.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'mentus'
npm ERR! notarget 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/202[4](https://github.com/PrimeDeviation/mentus/actions/runs/9434288137/job/25986215521?pr=177#step:4:5)-06-09T05_31_38_852Z-debug.log
Error: Process completed with exit code 1.

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     },
   "license": "MIT",
   "dependencies": {
-    "langchain": "^1.0.0"
+    "langchain": "^0.2.5"
   },
   "devDependencies": {}
 }

